pallets/dapp-staking/src/types.rs

```diff
@@ -1640,11 +1640,13 @@ pub enum TierThreshold {
     /// This value is constant and does not change between periods.
     FixedPercentage { required_percentage: Perbill },
     /// Entry into the tier is mandated by a percentage of the total issuance as staked funds.
-    /// The `percentage` is the amount required, which can change in-between periods, while `minimum_required_percentage`
-    /// is the minimum percentage that should not be reduced below.
+    /// This `percentage` can change between periods, but must stay within the defined
+    /// `minimum_required_percentage` and `maximum_possible_percentage`.
+    /// If `maximum_possible_percentage` is not specified, the default value used is 100%.
     DynamicPercentage {
         percentage: Perbill,
         minimum_required_percentage: Perbill,
+        maximum_possible_percentage: Option<Perbill>,
```

Comment:
Sorry for the repeated comment but please set this just as `Perbill` to simplify the code.